src/Task/PhpCsFixerV2.php

```diff
@@ -91,6 +86,8 @@ public function run(ContextInterface $context)
             return $this->runOnAllFiles($context, $arguments);
         }
 
+        $arguments->add('--path-mode=intersection');
```

Comment:
Hi @keradus,

The problem with override is that ignored files are being checked by php-cs-fixer which results in errors.
We cannot use the `Runner` for the same reason as mentioned above: some people are using the global installled php-cs-fixer, so we can't autoload the php-cs-fixer classes.

This is the problem we are trying to fix in this ticket, but there doesn't seem to be one fitting solution. As you mentioned, currently the best option is to handle errors with intersection and an empty finder.

Feel free to suggest any other solution if you think of something we didn't look at yet.